packages/@guardian/source-foundations/src/typography/renderers.tsx

```diff
@@ -0,0 +1,127 @@
+import {
```

Comment:
could we move these to something story-specific? (am i right that these are only used in stories? they'r enot actually part of `source-foundations`?

Reply:
Good call, moved to `.storybook`